app/components/site-chrome/template.hbs

```diff
@@ -82,7 +82,7 @@
     <div class="scrollable-body">
       <div class="{{if media.isLargeAndUp 'box box--nopad'}}">
 
-      <ul class="list list--noborder flush text--spacing text--semi {{unless media.isLargeAndUp 'list--inline list--spaced text--small' 'alignright text--medium '}}">
+      <ul class="list list--noborder flush {{unless media.isLargeAndUp 'list--inline list--spaced text--small' 'alignright '}}">
```

Comment:
Sorry, it was there before, so I had left it in there. I should have checked if it was there for a reason. Done!